db/migrate/20181015023430_create_promo_campaigns.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePromoCampaigns < ActiveRecord::Migration[5.2]
+  def change
+    create_table :promo_campaigns, id: :uuid,default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+      t.text :name
```

Comment:
You'll want to add a uniqueness index, https://stackoverflow.com/questions/45826033/how-do-i-create-a-unique-index-within-a-rails-create-table-migration

Reply:
I thought I added this.  Thanks